build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
 	alias(libs.plugins.sqldelight)
 	alias(libs.plugins.kotlin.jvm)
 	alias(libs.plugins.kotlin.plugin.serialization)
+
+	id("java-test-fixtures")
```

Comment:
Not needed? It's hard to see on mobile, but aren't the tests within this module?

Reply:
It's completely unnecessary, but text fixtures are cool and I finally have a project where I can use them (unnecessarily).